config/application.rb

```diff
@@ -84,8 +84,10 @@ class Application < Rails::Application
 
     # Disable ActionCable's request forgery protection
     # This is basically matching a set of allowed origins which is not good for us
-    # Our own origin-host forgery protection is implemented in lib/websocket_server.rb
-    Rails.application.config.action_cable.disable_request_forgery_protection = true
+    config.action_cable.disable_request_forgery_protection = false
+    # Matching the origin against the HOST header is much more convenient
+    config.action_cable.allow_same_origin_as_host = true
+    config.action_cable.mount_path = '/ws/notifications'
```

Comment:
verify:

This says that we have action cable mounting at `/ws/notifications`.
This will create the route necessary to handle this.
Now action cable works like it does for other rails apps. (It just happens to use a slightly different route)